src/translations/sqlb_ar_SA.ts

```diff
@@ -6567,20 +7070,24 @@ Hold %3Shift and click to jump there</source>
     </message>
     <message>
         <source>Ctrl+H</source>
-        <translation></translation>
+        <translation type="vanished">Ctrl+H</translation>
     </message>
     <message>
-        <location filename="../TableBrowser.cpp" line="42"/>
         <source>Filter in all columns</source>
-        <translation>رشّح كلّ الأعمدة</translation>
+        <translation type="vanished">رشّح كلّ الأعمدة</translation>
+    </message>
+    <message>
+        <location filename="../TableBrowser.cpp" line="42"/>
+        <source>Filter in any column</source>
+        <translation>رشّح أيّ عمود</translation>
     </message>
     <message>
         <location filename="../TableBrowser.cpp" line="102"/>
         <source>Ctrl+R</source>
-        <translation></translation>
+        <translation>Ctrl+R</translation>
```

Comment:
Shortcuts should not be translated.
See the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts) for further information.